### PR TITLE
Added DSA category

### LINKED ISSUE #1890: [Add] <Want to add DSA articles under subcategory "Articles">
### Site URL

https://medium.com/@mukhopadhyaypushan42/best-way-to-solve-dsa-problems-become-a-pro-ccba897469c1

### Description of the above provided site

Solving Data Structures and Algorithms (DSA) problems on platforms like LeetCode, InterviewBit, and GeeksforGeeks can indeed pose a considerable challenge. These platforms are renowned for refining your coding skills and preparing you for technical interviews. 

### Category

Other

### Sub Category

DSA

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by ayushrakesh**
## Fixes Issues 
Closes #1890 

## Changes proposed

Added DSA as a new category and subcategory - DSA-articles.

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
@rupali-codes @CBID2 
What do you think? After placement Prep, adding a dsa for this seems overkill. Since there is one for youtube as well.

Let me know what you think.

**Comment by ayushrakesh** [comment]
> Reference

Hello, I think it will be betterif we add new seperate section for DSA, although it is a huge topic and it needs many resources (videos+documentation+articles+other-stuffs like new trends).

**Comment by Anmol-Baranwal** [comment]
@ayushrakesh 
Let @CBID2 and @rupali-codes decide.

**Comment by ayushrakesh** [comment]
> I think that'll be helpful.

Ok thanks

**Comment by Anmol-Baranwal** [comment]
@ayushrakesh 
Why did you close this?

**Comment by ayushrakesh** [comment]
> @ayushrakesh Why did you close this?

I have reopened the pull request, please merge it.

File: database/DSA/DSAarticles.json
Comment (Anmol-Baranwal) [inline database/DSA/DSAarticles.json:3]:
```suggestion
      "name": "Mastering DSA Problem Solving: How to Become a Pro",
```

File: database/DSA/DSAarticles.json
Comment (Anmol-Baranwal) [inline database/DSA/DSAarticles.json:4]:
```suggestion
      "description": "This article offers valuable insights and guidance on excelling in Data Structures and Algorithms (DSA) problem-solving.",
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Please do these changes in this PR.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
Well done @ayushrakesh!😊 @rupali-codes and @Anmol-Baranwal, you're up! 😊

**Comment by ayushrakesh** [comment]
> Well done @ayushrakesh!😊 @rupali-codes you're up! 😊

So you have commit changes?,,I don't have to change more ?

**Comment by Anmol-Baranwal** [comment]
@ayushrakesh 
No, you don't have to make any changes. It is good and ready to merge if all the maintainers find it valid.

**Comment by rupali-codes** [comment]
@ayushrakesh please be patient, don't close the PR. It may take sometime for us to review the PRs.

**Pull request merged**
